coregistration/requirements.txt

```diff
@@ -1,2 +1 @@
 eo-learn-core
-opencv-contrib-python-headless>=4.1
```

Comment:
open-cv is not required anymore? this is a great victory!

Reply:
`opencv-python` it is still required, but not the `contirb`